config/initializers/assets.rb

```diff
@@ -3,6 +3,7 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 Rails.application.config.assets.precompile += %w( home.css )
+Rails.application.config.assets.precompile += %w( users.css )
```

Comment:
It's possible to do this in a single line:
`Rails.application.config.assets.precompile += %w(home.css  users.css)`